igvc_gazebo/nodes/ground_truth/ground_truth_republisher.cpp

```diff
@@ -67,7 +91,7 @@ void groundTruthCallback(const nav_msgs::Odometry::ConstPtr& msg)
   }
 }
 
-void utm_callback(const ros::TimerEvent& event, const tf::Transform& odom_to_utm)
+void GroundTruthRepublisher::utm_callback(const ros::TimerEvent &event, const tf::Transform &odom_to_utm)
 {
   static tf::TransformBroadcaster br;
   static tf::TransformListener tf_listener;
```

Comment:
Also probably a good idea to keep these in the header file. Would probably need to rename the broadcasters to differentiate them and the respective transform that they broadcast.